internal/aws.go

```diff
@@ -2,11 +2,11 @@ package internal
 
 import (
 	"context"
-	"fmt"
 	"log"
```

Comment:
This file still imports "log" and "fmt" packages. Make sure only the verbose package is used, otherwise it means that we have missed some lines where we are logging stuff.